sdk/search/azure-search/src/main/java/com/azure/search/models/CoordinateSystem.java

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * The EPSG:4326 is the coordinate system used for GeographyPoints in Azure Cognitive Search.
+ */
+@Fluent
+public class CoordinateSystem {
+    private static final String NAME_PROPERTY = "name";
+
+    @JsonProperty
+    private String type;
+
+    @JsonProperty
+    private Map<String, String> properties;
+
+    /**
+     * Creates a new instance for CoordinateSystem, with default values.
+     * The EPSG:4326 is the coordination system used for GeographyPoints in Azure Cognitive Search.
+     *
+     * @return a new instance of CoordinateSystem
+     */
+    public static CoordinateSystem create() {
+        Map<String, String> props = new HashMap<>();
+        props.put(NAME_PROPERTY, "EPSG:4326");
+        return new CoordinateSystem()
+            .setType(NAME_PROPERTY)
+            .setProperties(props);
+    }
+
+    /**
+     * Ensures that the values are valid as "crs" field in the GeoPoint for the Azure Cognitive Search service:
+     * \"crs\":{\"type\":\"name\",\"properties\":{\"name\":\"EPSG:4326\"}}
+     *
+     * @return true if valid, false if invalid
+     */
+    public boolean isValid() {
+        return StringUtils.equals(NAME_PROPERTY, type)
```

Comment:
How does `StringUtils.equals` differ from `CharSequence.compare == 0` or `Objects.equals`.

Reply:
this was just an implemetation decision to use StringUtils